boards/arc/emsdp/emsdp_em4.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: emsdp
```

Comment:
All the emsdp boards have the same identifier (emsdp), but these should be unique.
Please change into emsdp_em4 here, and _em6 etc. in the other board yaml files.

Reply:
already done.